stations.csv

```diff
@@ -64047,4 +64047,5 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 68188;Le Bouscat Sainte-Germaine;le-bouscat-sainte-germaine;;87738096;44.87090;-0.61400;;FR;Europe/Paris;f;f;f;t;t;f;FRKTO;;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68188;
 68189;Andria Sud;andria-sud;8355025;;41.224651;16.314623;29245;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8355025;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68189;
 68190;Ostarije;ostarije;7875111;;45.234444;15.296389;;HR;Europe/Zagreb;t;f;f;t;t;f;;;f;;f;7875111;f;;f;;f;;f;;f;;f;7875111;t;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68190;
-68191;Corato sud ospedale;corato-sud-ospedale;;;41.14550687319366;16.419894058269794;29255;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68191;
\ No newline at end of file
+68191;Corato sud ospedale;corato-sud-ospedale;;;41.14550687319366;16.419894058269794;29255;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68191;
+68192;Majadahond;majadahond;7110007;;40.47441400551557;-3.846237484654251;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68192;
```

Comment:
Should it be Majadahonda?
https://es.wikipedia.org/wiki/Majadahonda
https://en.wikipedia.org/wiki/Majadahonda

Reply:
Yes, actually it should be. Thank you!

Reply:
Changed the branch name to be accurate https://github.com/trainline-eu/stations/pull/1208